docs/gadget-devel/gadget-wasm-api-raw.md

```diff
@@ -445,3 +445,26 @@ Parameters:
 
 Return value:
 - 0 in case of success, 1 otherwise.
+
+### Syscalls
+
+#### `getSyscallName(id uint32) uint64`
+
+Get the syscall name for this syscall ID.
+
+Parameters:
+- `id` (u32): Syscall ID
+
+Return value:
+- (u64) A string containing the name of the syscall, in case the syscall is unknown, it returns "syscall_ID" with ID displayed as hexadecimal like strace.
+
+#### `getSyscallDeclaration(name uint64, pointer uint64) uint32`
+
+Get the syscall declaration for this syscall name.
+
+Parameters:
+- `name` (u64): Syscall name.
+- `pointer` (u64): A pointer to a syscallDeclaration structure, it is used to store the data instead of returning them.
```

Comment:
Can you add a link to this structure definition?

Reply:
A user reading this documentation won't have any clue of what "syscallDeclaration structure" is. I'm suggesting to add a link to the Github code defining it (it doesn't matter if the link is broken until we merge this PR), or copy and paste the definition here. The goal is to show the layout of the structure, different fields and their sizes.